init/MUOS/language/German.json

```diff
@@ -285,20 +285,20 @@
     "1080p + 60hz": "1080p + 60hz",
     "Sleep Suspend": "Ruhezustand Verzögerung",
     "Instant Shutdown": "Sofortiges Herunterfahren",
-    "Sleep 10s + Shutdown": "Ruhezustand 10s + Shutdown",
-    "Sleep 30s + Shutdown": "Ruhezustand 30s + Shutdown",
-    "Sleep 60s + Shutdown": "Ruhezustand 60s + Shutdown",
-    "Sleep 2m + Shutdown": "Ruhezustand 2m + Shutdown",
-    "Sleep 5m + Shutdown": "Ruhezustand 5m + Shutdown",
-    "Sleep 10m + Shutdown": "Ruhezustand 10m + Shutdown",
-    "Sleep 30m + Shutdown": "Ruhezustand 30m + Shutdown",
-    "Sleep 60m + Shutdown": "Ruhezustand 60m + Shutdown",
+    "Sleep 10s + Shutdown": "Ruhezustand 10s + Herunterfahren",
```

Comment:
This might have to be shortened as this string will be quite long and will overlap the right hand side label.

Reply:
We are currently discussing whether to use anglicisms instead for Shutdown / Standby / Sleep and so on to shorten the strings.